netmanager/src/assets/css/map-filter.css

```diff
@@ -38,19 +38,38 @@
 }
 
 .map-filter-searchButton {
-  width: 40px;
-  height: 36px;
+  width: 48px;
```

Comment:
Is this change part of this PR's scope? Could we be mixing up work of different branches/PRs? Just curious....awaiting your feedback.

Reply:
I checked out from a branch I thought had been merged, instead of checking out from staging.
I think let me delete this branch and re-push?